### PR TITLE
Just add some tests to have full coverage

### PR DESCRIPTION
**Pull request by grigouze**
Now it's 💯 % coverage, i don't if it's good or bad but it was easy :)

### CODE REVIEW

File: test_maya.py [hunk 1]
```diff
@@ -42,11 +45,30 @@ def test_dt_tz_naive():
...
+
+
+def test_print_date(capsys):
+    d = maya.when('11-17-11')
+
+    print(d)
+    out, err = capsys.readouterr()
+    assert out == '<MayaDT epoch=1321488000.0>\n'
+
+    assert type(d.__format__()) is datetime
```
Comment (timofurrer) [inline test_maya.py]: I don't think that's actually intended behavior. See #8

File: test_maya.py [hunk 2]
```diff
@@ -42,11 +45,30 @@ def test_dt_tz_naive():
     assert d1.hour - d2.hour == 5
 
 
-def test_random_date():
-    d = maya.when('11-17-11')
+def test_random_date(capsys):
```
Comment (timofurrer) [inline test_maya.py]: `capsys` was probably just for debugging?

Comment (grigouze) [inline test_maya.py]: No it's for capturing stdin and stout in pytest, see http://doc.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function

Comment (grigouze) [inline test_maya.py]: It's fixed. Thanks

File: test_maya.py (hunk 1 above)
Comment (grigouze) [inline test_maya.py]: yes you are right

**Comment by kennethreitz** [comment]
✨🍰✨

**Pull request merged**